app/src/main/java/com/github/sdp_begreen/begreen/viewModels/ConnectedUserViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.github.sdp_begreen.begreen.viewModels
+
+import android.graphics.Bitmap
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.github.sdp_begreen.begreen.models.User
+
+class ConnectedUserViewModel: ViewModel() {
```

Comment:
It's weird that this ViewModel is just holding state without doing anything else. For instance, it should probably be in charge of ensuring the picture is actually that of the user, instead of letting any other class mutate these properties independently?

Reply:
Yes you are right, I still have a lot to learn about `LiveData` and `ViewModel`. I did a bit of research, and I saw that in general the viewModel should store the `MutableLiveData` internaly and expose only a `immutableLiveData` to the world. I will implement that to ensure some sanity check on the picture that is added to ensure that it is actually related to this user.

Thank you very much for pointing this out !